public/scripts/countryData.js

```diff
@@ -0,0 +1,90 @@
+(function(module) {
```

Comment:
You have a lot of declarations here, which aren't totally bad for clarity, but you could dynamically create these `x.all = []` with a `.map()` or `.reduce()`.